packages/dev/sharedUiComponents/src/components/reactGraphSystem/NodeRenderer.tsx

```diff
@@ -0,0 +1,122 @@
+import type { ComponentType } from "react";
+import { useState, useEffect, useMemo } from "react";
+import { GraphContainer } from "shared-ui-components/components/reactGraphSystem/GraphContainer";
+import { GraphLine } from "shared-ui-components/components/reactGraphSystem/GraphLine";
+import { GraphNode } from "shared-ui-components/components/reactGraphSystem/GraphNode";
+import { GraphNodesContainer } from "shared-ui-components/components/reactGraphSystem/GraphNodesContainer";
+import { GraphLinesContainer } from "shared-ui-components/components/reactGraphSystem/GraphLinesContainer";
+import { GraphContextManager } from "shared-ui-components/components/reactGraphSystem/GraphContextManager";

```

Comment:
temporarily, if those could be relative it would be great.
There was an issue with the public publish process of this specific package, so use relative only for now.